docs/forms.md

```diff
@@ -296,4 +296,34 @@ OneBlink.Forms.validateForm(form)
   "isMultiPage": false,
   "isInfoPage": false 
 }
-```
\ No newline at end of file
+```
+
+## Generate Form Element
+
+`generateFormElement()` is a static method available on the forms class, used for both creating and validating a OneBlink Form Element.
+
+The method will set reasonable defaults for any values not passed to it, and validate ones that are against our Element Schema.
+
+### Example
+
+```javascript
+const element = {
+  "name": "my test element"
+}
+
+OneBlink.Forms.generateFormElement(element)
```

Comment:
This does not return a promise anymore right?

Reply:
Do we need to change the docs for `validateForm()` as well?

Reply:
Yeah my bad, missed these. Fixed now :)